src/calculateRentalCost.js

```diff
@@ -28,7 +28,14 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  let totalCost = 40 * days;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Same here, consider moving the discount value (50) to the variable and declaring them at the top of the function